src/lib/numbers.ts

```diff
@@ -320,7 +320,14 @@ export function applyFactor(value: BigNumber, factor: BigNumber) {
 }
 
 export function getBasisPoints(numerator: BigNumber, denominator: BigNumber) {
-  return numerator.mul(BASIS_POINTS_DIVISOR).div(denominator);
+  const result = numerator.mul(BASIS_POINTS_DIVISOR).div(denominator);
+  const remainder = numerator.mul(BASIS_POINTS_DIVISOR).mod(denominator);
+
+  if (!remainder.isZero()) {
+    return result.isNegative() ? result.sub(1) : result.add(1);
```

Comment:
is it always desirable behavior? we're using `getBasisPoints` in many places, i'm not sure if it is expected to update rounding logic for all of them

Reply:
Updated!